automated install/uninstall.sh

```diff
@@ -98,6 +98,10 @@ removePiholeFiles() {
         ${SUDO} cp -p /etc/systemd/resolved.conf.orig /etc/systemd/resolved.conf
         systemctl reload-or-restart systemd-resolved
     fi
+    if [[ -e /etc/systemd/resolved.conf.d/90-pi-hole-disable-stub-listener.conf ]]; then
+        ${SUDO} rm -f /etc/systemd/resolved.conf.d/90-pi-hole-disable-stub-listener.conf
+        systemctl reload-or-restart systemd-resolved
```

Comment:
Can you combine this and the check above to prevent `systemd-resolved` to restart twice in a row?

Reply:
Or use a variable to decide if the service needs to be restarted:

```sh
    # Restore Resolved
    local restart_service="false"
    if [[ -e /etc/systemd/resolved.conf.orig ]]; then
        ${SUDO} cp -p /etc/systemd/resolved.conf.orig /etc/systemd/resolved.conf
        restart_service="true"
    fi
    if [[ -e /etc/systemd/resolved.conf.d/90-pi-hole-disable-stub-listener.conf ]]; then
        ${SUDO} rm -f /etc/systemd/resolved.conf.d/90-pi-hole-disable-stub-listener.conf
        restart_service="true"
    fi

    if [[ "${restart_service}" == "true" ]]; then
        systemctl reload-or-restart systemd-resolved
    fi
```